jms/1.7/modules/ROOT/pages/jms-topic-subscription.adoc

```diff
@@ -39,12 +63,23 @@ The JMS Connector, *when configured with a JMS 2.0 connection*, allows you to de
 </jms:listener>
 ----
 
-With this configuration on two different `listener` instances, when a Message is published to the invoices destination it will be delivered only to one of the listeners and not the other.
 
+== Configure No Local Subscriptions
 
-== How to Ignore Self Published Messages
+There are cases in which you may not want to receive the published messages to a topic by the same application. To prevent the consumer to receive published messages to the topic by the same connection, configure the *Topic consumer* field to *No local*. Consumers configured as no local can not be part of a shared subscription.
```

Comment:
Yeah...I think you need to have a peer look at this whole topic, because I am lost.
My guess for this text is...:
```suggestion
If you want to limit the published messages available to a consumer to only those from outside connections, set *Topic consumer* to *No local*.  Consumers configured as such cannot be part of a shared subscription.
```